apps/services/user-notification/src/app/modules/notifications/notifications.service.ts

```diff
@@ -44,10 +46,9 @@ const ALLOWED_REPLACE_PROPS: Array<keyof HnippTemplate> = [
   'clickActionUrl',
 ]
 
-/**
- * Finds {{key}} in string
- */
-const ARG_REPLACE_REGEX = new RegExp(/{{[^{}]*}}/)
+type SenderOrganization = {
+  title: string | undefined
+}
```

Comment:
```suggestion
type SenderOrganization = {
  title?: string
}
```

Reply:
subtle difference - this is required to have the property title which can have a value of either string or undefined
title? would make the property optional